### PR TITLE
[DR] Import cluster via CLI

### PR DESCRIPTION
**Pull request by OdedViner**
Manual procedure:
https://url.corp.redhat.com/91f0590

### CODE REVIEW

**Review by ebenahar — COMMENTED** [review]
@OdedViner @shylesh as discussed, once this PR gets merged, let's use the CLI method instead of UI in the RDR production job

File: ocs_ci/ocs/acm/acm.py [hunk 1]
```diff
@@ -414,15 +417,59 @@ def import_clusters_with_acm():
...
+            auto_import_secret = templating.load_yaml(
+                "ocs_ci/templates/acm-deployment/auto-import-secret.yaml"
+            )
+            auto_import_secret["metadata"]["namespace"] = cluster[0]
+            auto_import_secret["stringData"]["kubeconfig"] = cluster[1]
+            auto_import_secret_obj = OCS(**auto_import_secret)
+            auto_import_secret_obj.apply(**auto_import_secret)
+
+            log.info("Wait managedcluster move to Available state")
+            ocp_obj = OCP(kind="managedcluster")
```
Comment (prsurve) [inline ocs_ci/ocs/acm/acm.py:457]: make managedcluster as constant

Comment (OdedViner) [reply ocs_ci/ocs/acm/acm.py]: done

File: ocs_ci/ocs/acm/acm.py [hunk 2]
```diff
@@ -414,15 +418,59 @@ def import_clusters_with_acm():
...
-        cluster_name=cluster_name_b,
-        kubeconfig_location=kubeconfig_b,
-    )
+        acm_nav.import_cluster(
+            cluster_name=cluster_name_b,
+            kubeconfig_location=kubeconfig_b,
+        )
+    else:
+        for cluster in clusters:
+            log.info(f"Create project {cluster[0]}")
```
Comment (am-agrawa) [inline ocs_ci/ocs/acm/acm.py:437]: when looping, why do we need list index to be defined to 0?

Comment (OdedViner) [reply ocs_ci/ocs/acm/acm.py:437]: on cluster, we have 2 items [cluster_name and kubeconfig]

File: ocs_ci/ocs/acm/acm.py [hunk 3]
```diff
@@ -414,15 +418,59 @@ def import_clusters_with_acm():
...
-    acm_nav.import_cluster(
-        cluster_name=cluster_name_b,
-        kubeconfig_location=kubeconfig_b,
-    )
+        acm_nav.import_cluster(
+            cluster_name=cluster_name_b,
+            kubeconfig_location=kubeconfig_b,
+        )
+    else:
+        for cluster in clusters:
```
Comment (shylesh) [inline ocs_ci/ocs/acm/acm.py:436]: @OdedViner , can  we move this entire 'else' case to a function, maybe something like ```import_clusters_via_cli``` or something?

File: ocs_ci/ocs/acm/acm.py (hunk 2 above)
Comment (shylesh) [inline ocs_ci/ocs/acm/acm.py:437]: @OdedViner , if we could print somewhere that "Importing clusters via CLI method" would be more readable

File: ocs_ci/ocs/acm/acm.py (hunk 3 above)
Comment (OdedViner) [reply ocs_ci/ocs/acm/acm.py:484]: done

**Review by petr-balogh — DISMISSED** [review]

**Review by petr-balogh — APPROVED** [review]

**Review by shylesh — APPROVED** [review]
LGTM

**Pull request merged**